importexport/musicxml/exportxml.cpp

```diff
@@ -2403,21 +2419,38 @@ void ExportMusicXml::chordAttributes(Chord* chord, Notations& notations, Technic
       // and finally the attributes whose elements are children of <technical>
       for (const Articulation* a : na) {
             auto sid = a->symId();
+            QString placement = "";
+            QString direction = "";
 
-            if (sid == SymId::stringsHarmonic) {
+            QString attr = QString("");
```

Comment:
@lvinken , remove initialization here too, right?

Reply:
Yes, "String attribute;" is sufficient